docs/2.0. APIs.md

```diff
@@ -62,8 +62,6 @@ Server Metadata.
     using the authorization code grant, as defined by [RFC 6749][RFC-6749].
 *   The `token_endpoint`, for example at **/token** - Used for requesting a token when using any of the chosen grant
     types, as defined by [RFC 6749][RFC-6749].
-*   The `revocation_endpoint`, for example at **/revoke** - Used as the revocation endpoint for revoking currently
-    active access tokens or refresh tokens, as defined by [RFC 7009][RFC-7009].
```

Comment:
In order to keep the list of the Authorization Endpoints in one place, I suggest keeping these lines and the reference for RFC 7009 here, but modifying like so:

```suggestion
*   When included in the [Server Metadata](#server-metadata-endpoint), the `revocation_endpoint`, for example at **/revoke** - Used as the revocation endpoint for revoking currently	
    active access tokens or refresh tokens, as defined by [RFC 7009][RFC-7009].
```